### PR TITLE
Release Teacher 1.13.1 41 

### PR DESCRIPTION
**Pull request by hermannakos**

### CODE REVIEW

**Comment by davidvarga93** [comment]
QA+1

**Review by balintbartok — APPROVED** [review]
Smoke test + 1

File: apps/student/src/main/res/layout/fragment_assignment_details.xml [hunk 1]
```diff
@@ -45,7 +45,8 @@
             android:layout_width="match_parent"
             android:layout_height="0dp"
             android:layout_weight="1"
-            app:layout_constraintTop_toBottomOf="@id/toolbar">
+            app:layout_constraintTop_toBottomOf="@id/toolbar"
+            app:layout_constraintBottom_toTopOf="@+id/submitButton">
```
Comment (tamaskozmer) [inline apps/student/src/main/res/layout/fragment_assignment_details.xml:49]: Why do we have this change here as well? I thought this only affects the student app.

**Review by tamaskozmer — APPROVED** [review]

**Comment by tamaskozmer** [comment]
Smoke test + 1

**Pull request merged**